### DISCUSSION TITLE
[Kontoverwaltung] Lehrerliste Lehrer/Schüler

### DISCUSSION THREAD

**Discussion by Schmarvinius** [SE-TINF22B2/G2-Schoolify#125] [User Stories]
## User Story:
Als Schüler/Lehrer möchte ich eine Liste meiner aller jetzigen und ehemaligen angestellten Lehrer habe, um bei einem Anliegen den bestimmten Lehrer kontaktieren zu können.

## Beschreibung
Eine Liste, in der alle ehemalig und aktuell angestellten Lehrer sind.

## Akzeptanz-Kriterien
- [ ] Liste an Lehrern wird angezeigt
- [ ] Liste an ehemaligen Lehrern wird angezeigt

Issues:
- #136 
- #137

**Comment by laralngn** [comment]
Nutzen fehlt

**Comment by laralngn** [comment]
Duplikat zu #130 ?
